R/helpers.R

```diff
@@ -26,6 +26,54 @@ groups_by_filter <- function(covariate, nbins, ties.method="random", seed=NULL){
 	as.factor(ceiling( rfs* nbins))
 }
 
+#' Stratify hypotheses based on increasing value of the covariate 
+#'
+#'  Hypotheses are stratified into nbins different strata of (approximately) equal size based on
+#' increasing value of the covariate
+#' extension of `groups_by_filter` for multidimensional covariates, per-covariate quantiles
```

Comment:
You are using `` ` `` to indicate code. This relies on markdown formatting which Roxygen does not used by default. You can turn it on by adding `Roxygen: list(markdown = TRUE)` to the `DESCRIPTION` file (see for example https://github.com/const-ae/glmGamPoi/blob/master/DESCRIPTION). However as Nikos seems to be rather using the explicit notation (i.e. `\code{\link{rank}}`), so I would recommend that you adopt the same style.

Reply:
Thank you, I changed it everywhere